build/package.yml

```diff
@@ -21,4 +21,11 @@ steps:
     inputs:
       pathtoPublish: '$(build.artifactStagingDirectory)/nupkgs'
       artifactName: 'nuget'
-      publishLocation: 'container'
\ No newline at end of file
+      publishLocation: 'container'
+
+  - task: PublishSymbols@2
+    displayName: 'Publish Symbols'
+    inputs:
+      searchPattern: '$(build.artifactStagingDirectory)\**\*.pdb'
```

Comment:
What is the default `<DebugType>` in msbuild for the new SDK format? If it's not portable, then we should probably change to `<DebugType>portable</DebugType>`

Reply:
I'm assuming it's portable because that's what gets built currently - will look for documentation.

Reply:
yes - https://github.com/dotnet/sdk/blob/8c81f32aafb7b46678444f5354bc73dcde75a664/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props#L76